### PR TITLE
chore: improve readablity

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### CODE REVIEW

File: bpf/profiling/network/args.h [hunk 1]
```diff
@@ -244,9 +244,11 @@ static __inline struct socket_buffer_reader_t* read_socket_data(struct sock_data
     if (size > MAX_PROTOCOL_SOCKET_READ_LENGTH) {
         size = MAX_PROTOCOL_SOCKET_READ_LENGTH;
     }
+    // use & to work around an issue in kernel verifier.
+    // See https://github.com/apache/skywalking/discussions/10273
```
Comment (wu-sheng) [inline bpf/profiling/network/args.h:248]: Please put a summary in the comments directly.
We don't recommend to put a link in codes. The issue could be edited or be updated later.

Comment (spacewander) [reply bpf/profiling/network/args.h]: @wu-sheng 
I replace this with the original issue, so people can get rid of the workaround once the original issue is resolved.

**Comment by mrproliu** [comment]
I think [there](https://github.com/apache/skywalking-rover/blob/main/bpf/profiling/network/netmonitor.c#L285) should also be an update.

**Review by mrproliu — APPROVED** [review]

**Comment by wu-sheng** [comment]
e2e fails, please recheck.

**Comment by wu-sheng** [comment]
@spacewander This PR had been open for months, are you still working on this?

**Comment by wu-sheng** [comment]
@mrproliu This change seem breaking something.

**Comment by spacewander** [comment]
> @spacewander This PR had been open for months, are you still working on this?

I still could not figure out why the e2e tests are not passed.

**Comment by mrproliu** [comment]
> > @spacewander This PR had been open for months, are you still working on this?
> 
> I still could not figure out why the e2e tests are not passed.

It looks like the eBPF verification failed. Have you tried to download the [E2E logs](https://github.com/apache/skywalking-rover/suites/11762446093/artifacts/613197130) or try to run it on your local machine?

**Comment by spacewander** [comment]
I tried to read the logs but didn't find out why. As for the setup the e2e environment, I am too busy recently so I didn't try it.

**Comment by spacewander** [comment]
I decide to close this PR first, as I am still busy working on other stuff. Maybe someone else can work on it and figure out why the tests are not passed.

**Pull request closed without merging**